public/dockerfiles/mongodb-kubernetes-database/1.2.0/ubi/Dockerfile

```diff
@@ -0,0 +1,85 @@
+ARG imagebase
+FROM ${imagebase} as base
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+
+
+LABEL name="MongoDB Kubernetes Database" \
+      version="1.2.0" \
+      summary="MongoDB Kubernetes Database Image" \
+      description="MongoDB Kubernetes Database Image" \
+      vendor="MongoDB" \
+      release="1" \
+      maintainer="support@mongodb.com"
+
+
+
+
+
```

Comment:
```suggestion


```

q: should we remove these unnecessary newlines? I see this in other files as well.

Reply:
https://github.com/mongodb/mongodb-kubernetes/pull/347#discussion_r2314422961